README.rst

```diff
@@ -0,0 +1,298 @@
+Intel® Media SDK
```

Comment:
Can you, please, cherry-pick 2 commits and not squash them into one from here: https://github.com/dvrogozh/MediaSDK/commits/docs2. I really believe that things needs to be done step by step. First change .md to .rst, then - add new info. Otherwise new info is obscured.

Reply:
Done